### PR TITLE
Add new airdrop status: Missed 😭

### LINKED ISSUE #8071: Add new airdrop status: Missed :sob:
## Abstract

For time limited airdrops that we have code for claiming we can actually figure out if the airdrop claiming was (unfortunately for the user) missed.

image1

## Task

- Add a new airdrop status: Misssed (frontend and backend)
- Check if the airdrop is time limited
- If there is the decoder for the airdrop and the event for claiming does not yet exist and events up to the time limit have been decoded then mark it as missed.
- Be sad :sob:

### PR DESCRIPTION
**Pull request by DaWe35**
Closes #8071

## Checklist

- [x] The PR modified the frontend, and updated the ~~[user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst)~~ API docs to reflect the changes.

Screenshot_20240627_131906
Screenshot_20240627_131924

### CODE REVIEW

File: frontend/app/src/pages/defi/airdrops/index.vue [hunk 1]
```diff
@@ -45,9 +46,12 @@ const rows = computed<(Airdrop & { index: number })[]>(() => {
   const data = filterByAddress(get(airdrops), addresses);
   return data.filter((airdrop) => {
     const currentStatus = get(status);
+    const currentTime = Date.now() / 1000;
```
Comment (lukicenturi) [inline frontend/app/src/pages/defi/airdrops/index.vue:49]: just small question, is the cutoff time saved in UTC
If yes, then I think you need to use utc time instead here

Comment (DaWe35) [reply frontend/app/src/pages/defi/airdrops/index.vue:49]: Hi, I believe Date.now() is UTC.  source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC

**Review by yabirgb — COMMENTED** [review]
backend looks okey

**Comment by yabirgb** [comment]
@DaWe35 Thanks for your contribution. Can you squash your commits to leave only one? Also can you add a changelog entry to `docs/changelog.rst`?

File: frontend/app/src/pages/defi/airdrops/index.vue (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/src/pages/defi/airdrops/index.vue:49]: oh yeah, you're right, cool then

**Review by lukicenturi — APPROVED** [review]
Frontend changes are fine

**Comment by DaWe35** [comment]
Squashed the commits on a separate branch: https://github.com/rotki/rotki/pull/8190

**Pull request closed without merging**